app/dashboard/models.py

```diff
@@ -1024,6 +1024,29 @@ def psave_interest(sender, instance, **kwargs):
         bounty.save()
 
 
+class Activity(models.Model):
+    """Represent Start work/Stop work event.
+
+    Attributes:
+        ACTIVITY_TYPES (list of tuples): The valid activity types.
+
+    """
+
+    ACTIVITY_TYPES = [
+        ('work_started', 'Work Started'),
+        ('work_stopped', 'Work Stopped'),
```

Comment:
`worker_approved`
`worker_rejected`
`worker_applied`

are all new activity types we might want to handle